Scripts/git/gen_pack_index.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+cat <<EOF
+<?xml version="1.0" encoding="UTF-8" ?>
+<index schemaVersion="1.0.0" xs:noNamespaceSchemaLocation="PackIndex.xsd" xmlns:xs="http://www.w3.org/2001/XMLSchema-instance">
+  <vendor>ARM</vendor>
+  <url>https://raw.githubusercontent.com/ARM-software/CMSIS-DSP/pack-index/</url>
```

Comment:
When I try this URL in a browser, I am getting:

400: Invalid request

I don't know what's this URL is useful for in the pdsc ?